### DISCUSSION TITLE
AE Randomizer for Thunderstone Quest?

### DISCUSSION THREAD

**Discussion by xfire008** [on3iro/aeons-end-randomizer#507] [General]
Hello, I'm an inexperienced programmer who spent some time retooling a randomizer for Thunderstone Quest. I absolutely love what you've come up with for AE and was wondering if it'd be possible to utilize this app for TQ. Our community has been clamoring for a modern app to substitute for the hundreds of randomizer cards. I'd be more than happy to prep the data files for all the cards and options and could work on the code as well with some instruction. 

My previous work is here: https://boardgamegeek.com/filepage/183294/thundermaster-tsq-intelligent-randomizer

If there's anyone who'd be willing to assist, I'd be very grateful. I think the AE backend would be superb for TQ! Cheers.

**Comment by on3iro** [comment]
Hi :)

Generally speaking it should be possible to strip down the randomizer to its basic functionality. (So only the randomizer and its corresponding settings remain). I am not familiar with Thunderstone Quest at all, so I have no idea how its market is typically being generated. 
You would also basically have to adjust the whole data model as it probably can't be translated 1 to 1.

In all honesty I think it would probably be way easier to start a fresh project and just get some inspiration from our code if necessary. I currently barely have any time to dedicate to projects like this, but I could answer a question here and there, if you need some guidance.

Kind regards

PS: there really is no such thing as an actual backend for our randomizer. Its all static content + indexedDB on the client side to preserve state. This was a deliberate decision, because we wanted to create a self contained progressive web app without the necessity for an online connection.

**Comment by Crowney1** [comment]
Hey there! It's awesome to see your enthusiasm for creating a randomizer for Thunderstone Quest. Your willingness to jump in and help with the data files and code is fantastic! As for using the AE randomizer for TQ, it's a cool idea. Why not [roll a dice](https://flipsimu.com/dice-roller/) and give it a shot? Who knows, it might just be the solution our community has been waiting for. Good luck with your project!
